go/vt/vtadmin/api.go

```diff
@@ -1867,16 +1861,18 @@ func (api *API) getTabletForAction(ctx context.Context, span trace.Span, action
 		go func(c *cluster.Cluster) {
 			defer wg.Done()
 
-			ts, err := c.GetTablets(ctx)
+			ts, err := c.FindTablets(ctx, func(t *vtadminpb.Tablet) bool {
```

Comment:
Would using `c.FindTablet` instead apply here? If so we might be able to 🔪 some lines.

Reply:
Ohh. Yeah the `found` var is what prompted the question. so that is gr8.